.github/workflows/_rust.yml

```diff
@@ -100,6 +99,7 @@ jobs:
       - uses: ./.github/actions/setup-rust
       - uses: ./.github/actions/setup-tauri
         timeout-minutes: 5
+        runtime: true
```

Comment:
```suggestion
        with:
          runtime: true
```

Reply:
if only somebody could invent something to catch errors before running the code.